### PR TITLE
SQ-27/Settings part 2: the abootioning 🇨🇦

### PR DESCRIPTION
**Pull request by rock3r**
This PR:
 * Uniforms the menu items on all homepage pages
 * Creates the About screen (remixed from the Zeplin designs for added ✨)
 * Adds the bloody NDK code style settings SO ANDROID STUDIO WILL STOP TRYING TO ADD THEM ಠ_ಠ hopefully 

aboot

TODO:
 * Edge cases
 * Sub-screens:
    * FOSS licences
    * ~About screen~
 * Use CoordinatorLayout (possibly with the header collapsing too, eventually, but for now mostly for the snackbars)
 * Photo border
 * ~Obtain cover photo (? it's not passed through by Firebase which is very annoying, probably not worth it)~ defo not worth it

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -71,4 +71,12 @@
   <string name="about_title">About</string>
   <string name="settings_account_not_logged_in">Not logged in</string>
 
+  <string name="about_label">About</string>
+  <string name="about_powered_by">is powered by</string>
+  <string name="squanchy">Squanchy</string>
```
Comment (fourlastor) [inline app/src/main/res/values/strings.xml]: can we make this untranslatable?

**Review by fourlastor — APPROVED** [review]
Dobry! Jeden komentarz!

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (rock3r) [reply app/src/main/res/values/strings.xml]: Untranslatable, eh?

Comment (fourlastor) [reply app/src/main/res/values/strings.xml]: Sorry.

Comment (rock3r) [reply app/src/main/res/values/strings.xml]: No, I'm sorry.

Comment (fourlastor) [reply app/src/main/res/values/strings.xml]: You're sorry, eh?

Comment (rock3r) [reply app/src/main/res/values/strings.xml]: b437dce5e25027a87287db3e373fe779149cf5f1

**Comment by fourlastor** [comment]
Shipit, eh?

**Pull request merged**